_inc/client/main.jsx

```diff
@@ -115,6 +118,9 @@ class Main extends React.Component {
 	}
 
 	componentDidUpdate( prevProps ) {
+		// Track page view on change only
+		prevProps.route.path !== this.props.route.path && this.props.isSiteConnected && analytics.tracks.recordEvent( 'jetpack_wpa_page_view', { path: this.props.route.path } );
```

Comment:
Not sure how we use the result of `!==` comparison

Reply:
After a bit of googling, I found this `!==/===` > `&&` which now makes sense for me.